krish98sai/geo-puzzle
# Rails Docker setup -> master

This PR puts a rails backend on docker.

To Test:
1. clone the repo
2. 'git checkout docker-setup'
3. 'docker-compose build'
4. 'docker-compose up'
5. go to 192.168.99.100
6. Make sure you see the "Yay you're on rails!"
7. 'docker-compose down' to take the docker containers down.
